### PR TITLE
SM-6706: WorkResponse now returns section74s

### PR DESCRIPTION
**Pull request by lukeca362**

### CODE REVIEW

File: docs/api-documentation/versions-and-changes/v3/changelog.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ title: API Versions and Changes
 This section lists any significant changes made to this document (and by extension, the API interfaces themselves) introduced by each recent and upcoming future release.
 {: .govuk-body}
 
+Version 3.8 (19/08/2021):
```
Comment (ccurran01) [inline docs/api-documentation/versions-and-changes/v3/changelog.md:11]: This should be added to the docs now 👍

**Review by ccurran01 — COMMENTED** [review]
Under the sequencing section there is a "Viewing Works and Permits" header which lists the properties returned on the works response

Could we update the V3 version of this to include Section 74s

File: docs/api-documentation/versions-and-changes/v3/changelog.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ Update Work API with the following changes:
 </ol>
 
 
+Updated Work API with the following changes:
+{: .govuk-body}
```
Comment (ccurran01) [inline docs/api-documentation/versions-and-changes/v3/changelog.md:25]: Believe these two can go as well

File: docs/api-documentation/versions-and-changes/v3/changelog.md [hunk 3]
```diff
@@ -20,6 +20,9 @@ Update Work API with the following changes:
   </li>
 </ol>
 
+<ol class="govuk-list govuk-list--bullet">
+  <li>SM-6706: <code>WorkResponse</code> has updated to return <code>section74s</code> property which is an array of <code>Section74SummaryResponse</code> responses.</li>
```
Comment (ccurran01) [inline docs/api-documentation/versions-and-changes/v3/changelog.md:24]: This should probably reflect the new limit of 50. 

This li should be part of the one above. Sorry, could really have made that a bit clearer

Comment (lukeca362) [reply docs/api-documentation/versions-and-changes/v3/changelog.md:24]: Oh yeah, that makes sense

**Review by ccurran01 — APPROVED** [review]

**Pull request merged**